src/Transport/Addressing/Partitioning/Strategies/FailOverNamespacePartitioning.cs

```diff
@@ -6,7 +6,7 @@
     using Transport.AzureServiceBus;
     using Settings;
 
-    public class FailOverNamespacePartitioning : INamespacePartitioningStrategy
+    public class FailOverNamespacePartitioning : INamespacePartitioningStrategy, ICacheableNamespacePartitioningStrategy
```

Comment:
If `ICacheableNamespacePartitioningStrategy` implements `INamespacePartitioningStrategy` like suggested above this and the following strategies can just implement that interface

Reply:
Personally, I like the expressiveness. It also aligns with other contracts we had to do a similar thing in v7.